login/routers/controllers/user_controllers/AuthentificationServices.js

```diff
@@ -0,0 +1,48 @@
+const sequelize = require('sequelize');
+
+const sequelizeOperators = sequelize.Op;
+const bcrypt = require('bcrypt');
+const userModel = require('../../../db/models/user_models/UserModel.js');
+const createUserAccount = require('./UserServices.js');
+const createUserResponce = require('../../../db/user_db/CreateUserResponce.js');
+
+
+function signUp(registrationRequest, response) {
+  userModel.user.findOne({
```

Comment:
Look closely, you're duplicating code here and in one other file as well regarding the user, if you can't find it quickly, ask me, I'll point it out.